html/Task_5/style.css

```diff
@@ -0,0 +1,222 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* Page title */
+.page-title {
+  margin: 0;
+  padding: 20px 0;
+  text-align: center;
+}
+
+.page-title--font {
+  font-family: "Kaushan Script", cursive;
+  font-size: 45px;
+}
+
+.page-title--border {
+  border: lightgray 1px solid;
+}
+
+/* Daily information bar */
+.info-bar {
+  min-height: 30px;
+  display: flex;
+  align-items: center;
+  padding: 0 20px;
+}
+
+.info-bar--background {
+  background: #f4f4f4;
+}
+
+.info-bar--border {
+  border: 1px solid lightgrey;
+}
+
+.info-bar__span {
+  display: flex;
+  justify-content: center;
+}
+
+.info-bar__temperature,
+.info-bar__actions {
+  flex: 1;
+}
+
+.info-bar__date {
+  flex: 2;
+}
+
+.info-bar__actions {
+  color: #8ea4e5;
+}
+
+.info-bar__link--color {
+  cursor: pointer;
+  text-decoration: none;
+  color: #8ea4e5;
+}
+
+/* Banner image */
+.banner {
+  width: 100%;
+  min-height: 300px;
+  max-height: 400px;
+}
+
+/* Layout changing buttons */
+.btn {
+  display: none;
+}
+
+.btn__label {
+  margin: 10px;
+  cursor: pointer;
+  padding: 5px;
+  border-radius: 60px;
+  background: #57a9ed;
+  position: fixed;
+  bottom: 20px;
+  left: 20px;
+}
+
+.btn__label--position {
+  position: fixed;
+}
+
+.btn__label--position-1 {
+  bottom: 20px;
```

Comment:
you already has bottom at btn__label class, why do you specify it for every selector?